install/kubernetes/cilium/charts/hubble-ui/templates/svc.yaml

```
@@ -10,4 +10,5 @@ spec:
     - name: http
       port: 80
       targetPort: 12000
-  type: ClusterIP
+  type: {{ .Values.service.serviceType | default "ClusterIP" }}
```

Can you move the default value ClusterIP into values..yaml intead?

And also rename serviceType to just `type` as service.serviceType is kind of repetitive now.

```suggestion
  type: {{ .Values.service.type }}
```

---

Agree